src/binary-search-tree.js

```diff
@@ -32,7 +56,16 @@ class BinarySearchTree {
   // again. Whatever floats your boat.
   /* eslint-disable global-require */
   breadthFirstForEach(cb) {
```

Comment:
I'm not sure what your logic here is. This implementation doesn't pass the test. Perhaps when you have time, refer to the solution repository or rewatch the solution lecture video to get a better idea of what a working solution looks like and _why_ it works.